# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnocl/rebuild_safe.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./src/install.tcl.orig	2005-11-09 18:26:32.212398624 -0300
+++ ./src/install.tcl	2005-11-09 18:35:19.788404016 -0300
@@ -20,17 +20,13 @@
 set destDir [file join $dir gnocl-$version]
 switch -- $what {
    "install"   {
-                  if { [file exists $destDir] } {
-                     puts "$destDir exists already. Aborting installation."
-                     exit -1
-                  }
                   puts "Creating $destDir"
-                  file mkdir $destDir
+                  file mkdir -p $destDir
                   set files [glob *.so]
                   lappend files pkgIndex.tcl
                   foreach file $files {
                      puts "Copying $file"
-                     file copy $file $destDir
+                     file copy -force $file $destDir
                   }
                }
    "test"      {
